tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -307,3 +307,248 @@ def test_get_replays_pagination(self):
+                user_name="a",
+                user_ip_address="127.0.0.1",
+                sdk_name="a",
+                sdk_version="a",
+                os_name="a",
+                os_version="a",
+                browser_name="a",
+                browser_version="a",
+                device_name="a",
+                device_brand="a",
+                device_family="a",
+                device_model="a",
+            )
+        )
+        self.store_replays(
+            mock_replay(
+                seq2_timestamp,
+                project.id,
+                replay2_id,
+                platform="a",
+                dist="a",
+                user_id="a",
+                user_email="a",
+                user_name="a",
+                user_ip_address="127.0.0.1",
+                sdk_name="a",
+                sdk_version="a",
+                os_name="a",
+                os_version="a",
+                browser_name="a",
+                browser_version="a",
+                device_name="a",
+                device_brand="a",
+                device_family="a",
+                device_model="a",
+            )
+        )
+
+        with self.feature(REPLAYS_FEATURES):
+            # Run all the queries individually to determine compliance.
+            queries = [
+                "platform",
+                "dist",
+                "duration",
+                "sdk.name",
+                "os.name",
+                "os.version",
+                "browser.name",
+                "browser.version",
+                "device.name",
+                "device.brand",
+                "device.family",
+                "device.model",
+                "user.id",
+                "user.name",
+                "user.email",
+            ]
+
+            for key in queries:
+                # Ascending
+                response = self.client.get(self.url + f"?sort={key}")
+                assert response.status_code == 200, key
+
+                r = response.json()
+                assert len(r["data"]) == 2, key
+                assert r["data"][0]["id"] == replay2_id, key
+                assert r["data"][1]["id"] == replay1_id, key
+
+                # Descending
+                response = self.client.get(self.url + f"?sort=-{key}")
+                assert response.status_code == 200, key
+
+                r = response.json()
+                assert len(r["data"]) == 2, key
+                assert r["data"][0]["id"] == replay1_id, key
+                assert r["data"][1]["id"] == replay2_id, key
+
+    def test_get_replays_filter_bad_field(self):
+        """Test replays conform to the interchange format."""
+        self.create_project(teams=[self.team])
+
+        with self.feature(REPLAYS_FEATURES):
+            response = self.client.get(self.url + "?query=xyz:a")
+            assert response.status_code == 400
+            assert b"xyz" in response.content
+
+    def test_get_replays_filter_bad_value(self):
```

Comment:
only other case i can think of is bad operator, but good stuff 👍🏼

Reply:
Yeah I can't figure out how to specify the operator...  Otherwise I would have.  I'll try wmak.